tests/test_core.py

```diff
@@ -444,6 +444,15 @@ def test_int53(self):
     self.emcc_args += ['-sDEFAULT_LIBRARY_FUNCS_TO_INCLUDE=[$convertI32PairToI53,$convertU32PairToI53,$readI53FromU64,$readI53FromI64,$writeI53ToI64,$writeI53ToI64Clamped,$writeI53ToU64Clamped,$writeI53ToI64Signaling,$writeI53ToU64Signaling]']
     self.do_core_test('test_int53.c', interleaved_output=False)
 
+  def test_int53_convertI32PairToI53Checked(self):
+    self.emcc_args += ['-sDEFAULT_LIBRARY_FUNCS_TO_INCLUDE=[$convertI32PairToI53Checked]']
+    if EMTEST_REBASELINE:
+      self.run_process([EMCC, test_file('core/test_convertI32PairToI53Checked.cpp'), '-o', Path('a.js'), '-DGENERATE_ANSWERS'] + self.emcc_args)
+      ret = self.run_process(config.NODE_JS + [Path('a.js')], stdout=PIPE).stdout
+      open(test_file('core/test_convertI32PairToI53Checked.out'), 'w').write(ret)
```

Comment:
Like in https://github.com/emscripten-core/emscripten/pull/16865#discussion_r864500912, it looks like `create_file` was not designed to write to the Emscripten tree, only in the temp directories that are cwd(?)

Reply:
Oh yes, in that case you can use `write_file(test_file('core/test_convertI32PairToI53Checked.out'), ret)` (`from tools.utils import write_file`). 

It avoids having to do the `with open() as f:` block that I would otherwise suggest here.

Reply:
check